client/dashboard/sites/monitoring-performance-card/index.tsx

```diff
@@ -77,17 +75,181 @@ export default function MonitoringPerformanceCard( {
 	site: Site;
 	timeRange: number;
 } ) {
-	const { formattedData, isLoading } = useSiteMetricsData( site.ID, timeRange );
+	const { requestsData, responseTimeData, isLoading } = useSiteMetricsData( site.ID, timeRange );
+
+	const data: SeriesData[] = [
+		{
+			label: __( 'Requests per minute' ),
+			data: requestsData || [],
+			options: {
+				gradient: {
+					from: '#3858E9',
+					to: '#3858E9',
+					fromOpacity: 0.2,
+					toOpacity: 0,
+				},
+				stroke: '#3858E9',
+				legendShapeStyle: {
+					color: '#3858E9',
+				},
+			},
+		},
+		{
+			label: __( 'Average response time (ms)' ),
+			data: responseTimeData || [],
+			options: {
+				gradient: {
+					from: '#5BA300',
+					to: '#5BA300',
+					fromOpacity: 0.2,
+					toOpacity: 0,
+				},
+				stroke: '#5BA300',
+				legendShapeStyle: {
+					color: '#5BA300',
+				},
+			},
+		},
+	];
+
+	const lessThanMediumViewport = useViewportMatch( 'medium', '<' );
+
+	let numTicks: undefined | number;
+	switch ( timeRange ) {
+		case 168:
+			numTicks = lessThanMediumViewport ? 3 : 7;
+			break;
+		case 72:
+			numTicks = lessThanMediumViewport ? 3 : 6;
+			break;
+		case 24:
+		case 6:
+			numTicks = lessThanMediumViewport ? 4 : 12;
+			break;
+	}
+
+	const xAxisOptions = {
+		tickFormat: ( date: string ) => {
+			const d = new Date( date );
+
+			if ( timeRange <= 24 ) {
+				return `${ d.getHours() }:${ d.getMinutes().toString().padStart( 2, '0' ) }`;
+			}
+
+			if ( timeRange > 72 || ( timeRange > 24 && lessThanMediumViewport ) ) {
+				return `${ d.toLocaleDateString() }`;
+			}
+
+			return `${ d.toLocaleDateString() } ${ d.getHours() }:${ d
+				.getMinutes()
+				.toString()
+				.padStart( 2, '0' ) }`;
+		},
+		numTicks: numTicks,
+	};
+
+	const getLegendIcon = ( key: string, isTooltip = false ) => {
+		const isLegendGlyph = key.startsWith( 'legend-glyph-' );
+		if ( isLegendGlyph ) {
+			key = key.replace( 'legend-glyph-', '' );
+		}
+
+		switch ( key ) {
+			case 'Requests per minute':
```

Comment:
Will this work with the translated version?

Reply:
Good point, it doesn't.
I'll see if I can find another way to make it work.